src/cmd_line/commands/set.ts

```diff
@@ -153,7 +153,8 @@ export class SetCommand extends ExCommand {
     this.operation = operation;
   }
 
-  async execute(vimState: VimState): Promise<void> {
+  // We allow vimState to be `null` so we can use this command for vimrc loading (where vimState is not available).
+  async execute(vimState: VimState | null): Promise<void> {
```

Comment:
This codebase prefers `undefined` to `null` when given the option

Reply:
done.